src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -241,33 +241,31 @@ const Workspaces = props => {
         const isSubscribed = workspace.userIsSubscribed
         workspace = workspace.attributes
         return (
-          <Link
+          <IndexCardItem
+            {...workspace}
             key={workspace.id}
-            to={`/workspaces/${workspace.id}`}
-            style={{ textDecoration: 'none' }}
+            onClick={() => props.onSelect(workspace.id)}
```

Comment:
Does the `useHistory` hook and `history.push` work here? 
https://reacttraining.com/react-router/web/api/Hooks/usehistory
Would avoid needing the extra prop

Reply:
oh wow, good idea! I'll try that